spec/intl.html

```diff
@@ -1676,20 +1676,63 @@ <h1>
+            <dd>
+              It performs the calendar-specific operation of determining which fields are mutually exclusive for the field given by _key_.
+              The returned List may be empty.
+            </dd>
           </dl>
+          <p>
+            Mutually exclusive means that the fields are alternate, potentially conflicting, ways of expressing the same data.
+            This operation is relevant for calendars which accept fields other than the standard set of ISO calendar fields, in order to implement the Temporal objects' `with()` methods, and `Temporal.Calendar.prototype.mergeFields()` in such a way that the result is free of ambiguity or conflicts.
+          </p>
+          <p>
+            For example, given a _calendar_ that uses eras, such as *"gregory"*, a _key_ of *"year"* would mutually exclude *"era"* and *"eraYear"*, and vice versa, because specifying *"era"* and *"eraYear"* is an alternate way of specifying *"year"* and the two could potentially conflict.
+            In addition to the ISO 8601 mutual exclusion of *"month"* and *"monthCode"*, a possible implementation might produce the following results when _calendar_ is *"gregory"*:
+          </p>
+          <emu-table id="table-calendarmutuallyexclusivekeys-example">
+            <emu-caption>Example results of CalendarMutuallyExclusiveKeys</emu-caption>
+            <table>
+              <thead>
+                <tr>
+                  <th>_key_</th>
+                  <th>Returned List</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>*"era"*</td>
+                  <td>« *"year*" »</td>
+                </tr>
+                <tr>
+                  <td>*"eraYear"*</td>
+                  <td>« *"year"* »</td>
+                </tr>
+                <tr>
+                  <td>*"year"*</td>
+                  <td>« *"era"*, *"eraYear"* »</td>
+                </tr>
+                <tr>
+                  <td>*"month"*</td>
+                  <td>« *"monthCode*" »</td>
+                </tr>
+                <tr>
+                  <td>*"monthCode"*</td>
+                  <td>« *"month"* »</td>
+                </tr>
+              </tbody>
+            </table>
+          </emu-table>
           <emu-note>
-            For example, if _fields_ contains *"year"* but _additionalFields_ contains *"era"* and *"eraYear"*, then the returned list must not include *"year"* in order to avoid ambiguity or conflict between different ways of specifying the year.
+            In calendars where eras do not start and end at year and month boundaries, note that the returned List should contain *"era"* and *"eraYear"* if _key_ is *"day"*, *"month"*, or *"monthCode"*, in addition to *"year"*, because it's possible for changing the day or month to cause a conflict with the era.
```

Comment:
```suggestion
            In calendars like Japanese where eras do not start and end at year and/or month boundaries, note that the returned List should contain *"era"* and *"eraYear"* if _key_ is *"day"*, *"month"*, or *"monthCode"*, in addition to *"year"*, because it's possible for changing the day or month to cause a conflict with the era.
```